velox/docs/functions/window.rst

```diff
@@ -0,0 +1,80 @@
+================
+Window functions
+================
+
+Window functions perform calculations across rows of the query result. They
```

Comment:
> perform calculations across rows of the query result

This statement is confusing as "query results" usually refers to the final results of the query, hence, no further processing is possible.